HISTORY.md

```diff
@@ -6,6 +6,8 @@
 
 #### API
 
+- Implement CMA-MAE archive thresholds (#256)
+  - Revive the old implementation of `add_single` removed in (#221)
```

Comment:
Can you also fix the tests such that they call `add_single` and `add` with single solution? We want to make sure we test both implementations, since we currently assume `add_single` calls `add` but that is no longer the case.